rhtap/update-deployment.sh

```diff
@@ -52,7 +52,9 @@ function patch-gitops() {
 	yq e -i "${IMAGE_PATH} |= \"${PARAM_IMAGE}\"" "${deployment_patch_filepath}"
 
 	git add .
-	git commit -m "Update '${component_name}' component image to: ${PARAM_IMAGE}"
+	GIT_COMMITTER_NAME="${_GIT_NAME}" GIT_COMMITTER_EMAIL="${_GIT_EMAIL}" \
+	GIT_AUTHOR_NAME="${_GIT_NAME}" GIT_AUTHOR_EMAIL="${_GIT_EMAIL}" \
+	  git commit -m "Update '${component_name}' component image to: ${PARAM_IMAGE}"
```

Comment:
There is an --author flag that could be used for the author, but env vars is the only way to do it for the committer afaict.